node/overseer/src/lib.rs

```diff
@@ -1130,26 +1130,34 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
-			channel_fill_level_from_overseer: prometheus::register(
+			from_overseer_channel_queue_sizes: prometheus::register(
 				prometheus::Histogram::with_opts(
 					prometheus::HistogramOpts {
 						common_opts: prometheus::Opts::new(
-							"overseer_channel_fill_level_from_overseer",
+							"parachain_from_overseer_channel_queue_sizes",
 							"Number of elements sitting in the channel waiting to be processed.",
 						),
-						buckets: prometheus::exponential_buckets(0.00001_f64, 2_f64, (CHANNEL_CAPACITY as f64).log2().ceil() as usize).expect("inputs are within documented range; qed"),
+						buckets: prometheus::exponential_buckets(
+							0.00001_f64,
```

Comment:
I realize this is just a formatting, but I just realized: channel fill size is an integer, so any buckets between 0 and 1 are wasted.

```suggestion
							1_f64,
```